### PR TITLE
Add support and testing for Django 2.1

### PR DESCRIPTION
**Pull request by jdufresne**

### CODE REVIEW

**Comment by huangsam** [comment]
Is there anything this PR is waiting on before it gets merged? This is useful for folks who want to keep their Django package up-to-date.

File: CHANGES.txt [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+UNRELEASED
+----------
+
+- Add support and testing for Django 2.0. No actual code changes were required.
```
Comment (rpkilby) [inline CHANGES.txt]: `2.0` => `2.1` ?

**Comment by jdufresne** [comment]
Thanks for catching that. I updated the Changelog.

**Comment by niwinz** [comment]
Thanks!

**Pull request merged**